apps/meteor/app/api/server/lib/maybeMigrateLivechatRoom.ts

```diff
@@ -0,0 +1,28 @@
+import type { IOmnichannelRoom, IRoom } from '@rocket.chat/core-typings';
+import { Rooms } from '@rocket.chat/models';
+import type { FindOptions } from 'mongodb';
+
+import { migrateVisitorIfMissingContact } from '../../../livechat/server/lib/Contacts';
+
+export async function maybeMigrateLivechatRoom(room: IRoom | null, options: FindOptions<IRoom> = {}): Promise<IRoom | null> {
+	if (room?.t !== 'l') {
```

Comment:
Use isOmnichannelRoom typecheck so you can avoid the typecast below as well

Reply:
Done!